pip/baseparser.py

```diff
@@ -132,7 +132,7 @@ def __init__(self, *args, **kwargs):
         self.name = kwargs.pop('name')
 
         isolated = kwargs.pop("isolated", False)
-        self.config = Configuration(isolated)
+        self._configuration_obj = Configuration(isolated)
```

Comment:
Agreed. It's exactly why I put that there.

I had changed it to prevent collisions if I write that while cracking out the implementation because I don't use `_obj` suffix either. Forgot to revert that change.